### DISCUSSION TITLE
Qwen2-VL Training

### DISCUSSION THREAD

**Discussion by rburchcp** [roboflow/multimodal-maestro#87] [Ideas]
Hi, Qwen2-VL is mentioned in the README but there does not appear to be any code in the repo that allows us to use Qwen2-VL in training or inference. My understanding is that Qwen2-VL is the only model among the three mentioned that works on videos, and that's primarily why I'm interested in that model. Do y'all plan on introducing code soon that allows Qwen2-VL training on videos and associated descriptions?

**Comment by onuralpszr** [comment]
@rburchcp Maestro 1.0.0 now has Qwen2 fine tuning you can check out documentation and google collab 

https://maestro.roboflow.com/latest/#__tabbed_1_3
https://colab.research.google.com/github/roboflow/maestro/blob/develop/cookbooks/maestro_qwen2_5_vl_json_extraction.ipynb

I hope those helps
